universe/Effect.h

```diff
@@ -1123,7 +1123,7 @@ class FO_COMMON_API GenerateSitRepMessage : public EffectBase {
     const std::string& Icon() const
     { return m_icon; }
 
-    const auto& MessageParameters() const
+    auto MessageParameters() -> const std::vector<std::pair<std::string, ValueRef::ValueRefBase<std::string>*>>&
     { return m_message_parameters; }
```

Comment:
I just wanted to be rid of the unnecessary and long return type, and my first attempt with just auto as the return type didnt work in GCC. The error message led to me trying this syntax.